### ISSUE TITLE
Applied Hyper parameter tuning for XG Boost, Hyperparameters are N estimators, learning rate, max depth

### ISSUE THREAD

**Issue by sreedharjinka** [sreedharjinka/Thyroid-DetectionG97#10]

**Comment by Dhruv69408012** [comment]
The xgboost model had various parameters which could be tuned for better performance the major parameters being n_estimators, learning rate and max_depth. n_estimators signifies the number of estimators making the prediction this was set to 100 to avoid any unnecessary noise. The learning rate was set to the default value of 0.1 for proper and slow pace of learning and the max_depth was set to 5
